package.json

```diff
@@ -58,32 +58,32 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@lingui/cli": "^4.7.0",
-    "@lingui/conf": "^4.7.0",
+    "@lingui/cli": "^4.11.2",
```

Comment:
do we want to keep the ^ for `@lingui/cli` and `@lingui/conf`?

Reply:
Cleanup up now 👍